### PR TITLE
Synthetics can metabolize horny chems

### PR DESCRIPTION
**Pull request by CydiaButt13**
I WILL FIX IT WITH MY OWN 2 HANDS part 1
## About The Pull Request

Adds REAGENT_SYNTHETIC and REAGENT_ORGANIC flags to every horny chem.
## Why It's Good For The Game

more erp play for robots
## Proof Of Testing
Compiled and ran. Sizes go over previous max.

## Changelog
:cl:
add: REAGENT_SYNTHETIC and REAGENT_ORGANIC to all horny chems, allowing synthetic carbons to use them
:cl:

### CODE REVIEW

**Comment by SapphoQueer** [comment]
Not sure about the size cap, maybe limit it to dorms/ghost cafe?

**Comment by Arturlang** [comment]
Separate the size cap removal to a separate pr please, thats a little iffy if its wanted

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/breast_milk.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	taste_description = "warm and creamy"
 	color = "#ffffffff"
 	reagent_state = LIQUID
+	process_flags = REAGENT_ORGANIC | REAGENT_SYNTHETIC
```
Comment (Arturlang) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/breast_milk.dm:7]: Do the non-ERP chemicals even do anything for synthetics? If not there's no point to giving them the process flag

**Review by Arturlang — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/breast_milk.dm (hunk 1 above)
Comment (thegrb93) [reply modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/breast_milk.dm:7]: yes, they have an arousal stat

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/cum.dm [hunk 2]
```diff
@@ -23,6 +24,7 @@
 	taste_description = "musky and salty"
 	color = "#ffffffff"
 	reagent_state = LIQUID
+	process_flags = REAGENT_ORGANIC | REAGENT_SYNTHETIC
```
Comment (Arturlang) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/cum.dm:27]: Remove these and it'll be good

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/cum.dm [hunk 3]
```diff
@@ -4,6 +4,7 @@
 	taste_description = "astringent and sweetish"
 	color = "#ffffffb0"
 	reagent_state = LIQUID
+	process_flags = REAGENT_ORGANIC | REAGENT_SYNTHETIC
```
Comment (Arturlang) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/cum.dm:7]: Remove these and it'll be good

**Review by Arturlang — CHANGES_REQUESTED** [review]

**Pull request merged**